### PR TITLE
Parallax*Fragment; support multiple parallax-layouts

### PR DESCRIPTION
**Pull request by meierjan**
While customising the layout I faced an issue that only one parralax layout wrapper is supported per slide. I fixed that here.

### CODE REVIEW

File: library/src/main/java/com/heinrichreimersoftware/materialintro/view/parallax/ParallaxFragment.java [hunk 1]
```diff
@@ -7,38 +7,45 @@
...
+import java.util.Collections;
 import java.util.LinkedList;
+import java.util.List;
 import java.util.Queue;
 
 public class ParallaxFragment extends Fragment implements Parallaxable {
-    @Nullable
-    private Parallaxable parallaxLayout;
+
+    private List<Parallaxable> parallaxableChildren = Collections.emptyList();
```
Comment (heinrichreimer) [inline library/src/main/java/com/heinrichreimersoftware/materialintro/view/parallax/ParallaxFragment.java]: Attribute should be final as it's never changing.

File: library/src/main/java/com/heinrichreimersoftware/materialintro/view/parallax/ParallaxSlideFragment.java [hunk 2]
```diff
@@ -2,44 +2,52 @@
...
+import java.util.Collections;
 import java.util.LinkedList;
+import java.util.List;
 import java.util.Queue;
 
 public class ParallaxSlideFragment extends SlideFragment implements Parallaxable {
-    @Nullable
-    private Parallaxable parallaxLayout;
+
+    private List<Parallaxable> parallaxableChildren = Collections.emptyList();
```
Comment (heinrichreimer) [inline library/src/main/java/com/heinrichreimersoftware/materialintro/view/parallax/ParallaxSlideFragment.java]: Attribute should be final as it's never changing.

**Comment by heinrichreimer** [comment]
Thanks for the PR. Looks great and material-intro should definitely support that!

**Comment by meierjan** [comment]
Sorry I was camping during the weekend. Anything I can do?

**Comment by meierjan** [comment]
Hey Heinrich,
one more thing that bothered me (but I was too lazy) was that the code is duplicated. I can refactor this out into a Util function if you want me to.

**Comment by heinrichreimer** [comment]
@meierjan Awesome idea!

**Comment by meierjan** [comment]
having is setOffset() in the util function as well is the consequent result

**Pull request merged**